Library/Homebrew/formula.rb

```diff
@@ -307,6 +308,12 @@ def any_version_installed?
   end
 
   # @private
+  # If any old version of {Formula} is installed.
```

Comment:
Ok, I think I see what this is doing now but it's not super obvious for the name. Maybe `upgradable_version_installed?` or something?